src/protocols/ssh/settings.c

```diff
@@ -425,6 +425,10 @@ guac_ssh_settings* guac_ssh_parse_args(guac_user* user,
     settings->timezone =
         guac_user_parse_args_string(user, GUAC_SSH_CLIENT_ARGS, argv,
                 IDX_TIMEZONE, NULL);
+    
+    /* If timezone not explicitly set, try to pull from tunnel */
```

Comment:
Same here as with RDP - guacd doesn't know about the tunnel.

Reply:
Fixed.